portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,9 +14,9 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.classes.Comment;
+
 import java.io.IOException;
-<<<<<<< HEAD
-=======
```

Comment:
Please squash this commit with the one introducing the conflicts, so the history in master doesn't contain invalid code.